pkgs/build-support/mkTextileLoader.nix

```
@@ -16,6 +16,9 @@
 let
   lib_lock = lib.importJSON ./libraries.json;
   fetchedLibraries = lib.forEach libraries (l: fetchurl lib_lock.${l});
+  asmVersion = builtins.head (builtins.head (
+    builtins.filter (v: v!=null) (builtins.map (builtins.match "org\\.ow2\\.asm:asm:([\.0-9]+)") libraries)
+  ));
```

```suggestion
let
  inherit (builtins) head filter map match;
  
  lib_lock = lib.importJSON ./libraries.json;
  fetchedLibraries = lib.forEach libraries (l: fetchurl lib_lock.${l});
  asmVersion = head (head (filter (v: v!=null) (map (match "org\\.ow2\\.asm:asm:([\.0-9]+)") libraries)));
```

Bit of cleanup, all of the `builtins` was just noise.